helm/idol-qms/values.yaml

```diff
@@ -45,6 +50,22 @@ additionalVolumes: []
 # -- Additional PodSpec VolumeMount (see https://kubernetes.io/docs/reference/kubernetes-api/workload-resources/pod-v1/#volumes-1)
 additionalVolumeMounts: []
 
+# -- Optional configMap name holding extra environnment variables for container
+envConfigMap: ""
+
+# -- Optional PodSecurityContext (see https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.29/#podsecuritycontext-v1-core)
+podSecurityContext:
+  # -- enable PodSecurityContext. Setting to false omits.
+  enabled: false
+  fsGroup: 0
+  runAsUser: 1000
+  runAsGroup: 0
```

Comment:
Do any of these want docs comments either? Doesn't look like this was done in previous upgrades - not sure if that's intentional or not?

Reply:
The parent `podSecurityContext` comment points to https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.29/#podsecuritycontext-v1-core which covers all the elements — is that sufficient?

Reply:
Ah fair, yeah if there's aren't internal variables then pointing to the kubernetes docs makes more sense.